lib/db.js

```diff
@@ -211,8 +220,24 @@ module.exports = (
   }
 
   DB.prototype.sessions = function (uid) {
+    const promises = [
+      this.pool.get('/account/' + uid + '/sessions')
+    ]
+
+    if (this.redis) {
+      promises.push(this.redis.getAsync(uid))
+    }
+
     log.trace({ op: 'DB.sessions', uid: uid })
-    return this.pool.get('/account/' + uid + '/sessions')
+    return P.all(promises)
+      .then(results => {
+        const tokens = results[0]
+        const redisTokens = results[1] ? JSON.parse(results[1]) : []
+        return tokens.map(function (sessionToken) {
+          const redisToken = redisTokens.find(tok => tok.tokenId === sessionToken.tokenId)
+          return redisToken || sessionToken
```

Comment:
Yep, let's make it so the same properties are always returned, regardless of whether the object is from Redis or MySQL. Any properties that aren't in Redis should come from MySQL instead.